KM4K.py

```diff
@@ -56,15 +59,31 @@ def read_nfc():
                 tag.sys = 3
                 idm = binascii.hexlify(tag.idm)
                 return idm
-
+            
+def check_card_manager(idm):
+    url = "https://card.ueckoken.club/api/card/verify"
+    payload = json.dumps({
+      "idm": idm
+    })
+    headers = {
+      'x-api-key': os.environ["API_KEY"],
+      'Content-Type': 'application/json'
+    }
+    response = requests.request("POST", url, headers=headers, data=payload)
+    status = json.loads(response.text)
+    if(status['verified']!= None and status['verified'] == True):
```

Comment:
[nits]
```suggestion
    if status['verified'] is not None and status['verified']:
```
って書き変えると良いと思います。
Noneに対する比較は`==`ではなく`is`を用いて行います。